Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -458,6 +483,7 @@ namespace gdjs {
         }
 
         // Delete the body
+        this._body.gdjsAssociatedBehavior = null;
         this._sharedData.world.DestroyBody(this._body);
         this._body = null;
```

Comment:
This is the only actual change.

Reply:
I reverted it because it makes a test fail but it's probably the test that should be fixed.